Branch-SDK/src/io/branch/referral/ServerRequestCreateUrl.java

```diff
@@ -119,6 +119,17 @@ public void onRequestSucceeded(ServerResponse resp, Branch branch) {
         }
     }
 
+    /**
+     * Calls the callback with the URL. This should be called on finding an existing url
+     * up on trying to create a URL asynchronously
+     * @param url
+     */
+    public void onUrlAvailable(String url){
```

Comment:
it `ServerRequestCreateUrl.isAsync()` used to determine whether this is a synchronous request